jazz_environment-event-handler/test/test.js

```diff
@@ -181,11 +277,169 @@ describe('jazz environment handler tests: ', () => {
+		testPayloads.getEnvironmentLogicalId.statusCode = 400;
+		let requestPromiseStub = sinon.stub(request, "Request", (obj) => {
+			return obj.callback(null, testPayloads.getEnvironmentLogicalId, JSON.stringify(testPayloads.getEnvironmentLogicalId.body));
+		});
+
+		index.getEnvironmentLogicalId(environmentPayload, configData, authToken)
+		.catch((res) => {
+			sinon.assert.calledOnce(requestPromiseStub);
+			requestPromiseStub.restore();
+			testPayloads.getEnvironmentLogicalId.statusCode = statusCode
+			expect(res.error).to.eql('Could not get environment Id for service and domain');
+		});
+	});
+
+	it('Verify processEventUpdateEnvironment rejects with statusCode of 400', () => {
+		let environmentPayload = testPayloads.environmentPayload;
+		environmentPayload.friendlyName = "FriendlyName";
+		environmentPayload.logicalId = "6knr9d33tt-dev";
+
+		let requestPromiseStub = sinon.stub(request, "Request", (obj) => {
+			return obj.callback(null, testPayloads.processEventUpdateEnvironmentError, JSON.stringify(testPayloads.processEventUpdateEnvironmentError.body));
+		});
+
+		index.processEventUpdateEnvironment(environmentPayload, configData, authToken)
+		.catch(res => {
+			sinon.assert.calledOnce(requestPromiseStub);
+			requestPromiseStub.restore();
+			expect(res.details).to.eql('Error');
+		})
+	});
+
+	it('Verify processEventCreateBranch rejects with statusCode of 400', () => {
+		let environmentPayload = testPayloads.environmentPayload;
+		environmentPayload.logicalId = "6knr9d33tt-dev";
+
+		let requestPromiseStub = sinon.stub(request, "Request", (obj) => {
+			return obj.callback(null, testPayloads.createBranchError, JSON.stringify(testPayloads.createBranchError.body));
+		});
+
+		index.processEventCreateBranch(environmentPayload, configData, authToken)
+		.catch(res => {
+			sinon.assert.calledOnce(requestPromiseStub);
+			requestPromiseStub.restore();
+			expect(res.details).to.eql('error');
+		})
+	});
+
+	it('Verify processEventInitialCommit rejects with statusCode of 400', () => {
+		let environmentPayload = testPayloads.environmentPayload;
+
+		let requestPromiseStub = sinon.stub(request, "Request", (obj) => {
+			return obj.callback(null, testPayloads.processEventInitialCommitError, JSON.stringify(testPayloads.processEventInitialCommitError.body));
+		});
+
+		index.processEventInitialCommit(environmentPayload, configData, authToken)
+		.catch(res => {
+			sinon.assert.calledTwice(requestPromiseStub);
+			requestPromiseStub.restore();
+			expect(res.details).to.eql('error');
+		});
+	});
+
+	it('Verify processEventInitialCommit rejects when physical id is different', () => {
+		let environmentPayload = testPayloads.environmentPayload;
+		environmentPayload.physical_id = "physicalId";
+		
+		index.processEventInitialCommit(environmentPayload, configData, authToken)
+		.catch(res => {
+			expect(res).to.eql('INITIAL_COMMIT event should be triggered by a master commit. physical_id is physicalId');
```

Comment:
I would use the base condition variable in the expected result. So in this case I would append to the expected result as in  
'INITIAL_COMMIT event should be triggered by a master commit. physical_id is ' +  environmentPayload.physical_id